it/src/test/java/com/linecorp/centraldogma/it/mirror/git/LocalToRemoteGitMirrorTest.java

```diff
@@ -349,6 +350,14 @@ void cannotMirrorInternalRepositories(String localRepo) {
                 .hasMessageContaining("invalid localRepo:");
     }
 
+    @Test
+    void localToRemoteEmptyMessage() throws Exception {
+        pushMirrorSettings("/local", "/remote", null);
+        final RevCommit call = git.commit().setMessage("").call();
+        mirroringService.mirror().join();
+        assertThat(call.getFullMessage()).isEmpty();
+    }
```

Comment:
We're implementing this for remote-to-local cases only. Please remove this.

Reply:
Removed